rio/src/screen/mouse.rs

```diff
@@ -53,6 +54,18 @@ impl Default for Mouse {
     }
 }
 
+impl Mouse {
+    pub fn new(config: &Rc<rio_config::Config>) -> Self {
```

Comment:
since only multiplier is configurable and the rest of the values have a default, is better to just receive multiplier instead of the reference counter reference of config.

```rust
pub fn new(multiplier: f64) -> Self {
```

Reply:
editted .  f9d755f41fbfa3d16fdfbd7a25506f0b099b7cfb